src/code/z_sub_s.c

```diff
@@ -688,9 +872,54 @@ s32 SubS_FillCutscenesList(Actor* actor, s16 cutscenes[], s16 numCutscenes) {
     return i;
 }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013E4B0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013E5CC.s")
+/**
+ * Computes a plane based on a point on the plane, a unit vector and two angles
+ *
+ * @param[in] point a point on the plane
+ * @param[in] unitVec the unit vector rotated that becomes the plane's normal
+ * @param[in] rot the angles to rotate with, uses just the x and y components
+ * @param[out] plane the computed plane
+ *
+ * Notes:
+ *  The unit input vector is expected to already be normalized (only uses are with the z unit vector)
+ *  The plane paramaters are of form `ax + by + cz + d = 0` where `a,b,c` are the planes normal vector
```

Comment:
This last part should probably go in the header with the `Plane` struct, with a note about normalisation... although I haven't checked every function that uses it, it's almost certainly assumed.